src/components/spotlight/spotlight.html.twig

```diff
@@ -0,0 +1,146 @@
+{% apply spaceless %}
+
+{#
+  Parameters:
+  - "color_mode" (string) (default: '') Name of the color mode
+  - "font_size" (string) (default: 'l') Size of the title (can be 'm', 'l')
+  - "title" (string) (default: '') Title of the spotlight
+  - "path" (string) (default: '') Link url
+  - "header" (string) (default: '') Header over the spotlight
+  - "picture" (object) (default: {}): Image for the spotlight, following ECL Picture structure
+  - "has_anchor" (boolean) (default: true): Should the visual anchor be displayed?
+  - "video" (object) (default: {}): Video for the spotlight
+    - "poster" (string) (default: ''): Image to be used as the video placeholder 
+    - "sources" (array) (default: []): format: [
+      {
+        "src" (string) (default: ''),
+        "type" (string) (default: ''),
+      },
+      ...
+    ],
+    - "tracks" (array) (default: []): format: [
+      {
+        "src" (string) (default: ''),
+        "kind" (string) (default: ''),
+        "src_lang" (string) (default: ''),
+        "label" (string) (default: ''),
+      },
+      ...
+    ],
+  - "sr_play" (string) (default: '') screen reader label for the play button
+  - "sr_pause" (string) (default: '') screen reader label for the pause button
+  - "credit" (string) (default: '') Credit for the image
+  - "full_width" (boolean) (default: false) Full width spotlight (extend outside the grid)
+  - "extra_classes" (string) (default: '')
+  - "extra_attributes" (optional) (array) (default: [])
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (optional) (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-spotlight' %}
+{% set _extra_attributes = '' %}
+{% set _color_mode = color_mode|default('') %}
+{% set _picture = picture|default({}) %}
+{% set _video = video|default({}) %}
+{% set _font_size = font_size|default('m') %}
+{% set _full_width = full_width|default(false) %}
+{% set _header = header|default('') %}
+{% set _title = title|default('') %}
+{% set _path = path|default('') %}
+{% set _credit = credit|default('') %}
+{% set _has_anchor = has_anchor|default(true) %}
```

Comment:
this needs to use the null coalescence operator, otherwise it will always be true
```suggestion
{% set _has_anchor = has_anchor ?? true %}
```